config_example.sh

```diff
@@ -57,6 +57,9 @@
 #
 #export CAPBM_RUN_LOCAL=true
 
+#Set Kubenetes cluster 
```

Comment:
Can you add a comment about what the choices are?

default: `minikube`, also supported: `kinder` ?

Reply:
Done

Reply:
looks like an addition is still needed to clarify which one is the default